jmix-flowui/flowui/src/main/java/io/jmix/flowui/facet/settings/component/binder/AbstractGridSettingsBinder.java

```diff
@@ -104,6 +111,16 @@ public void applyDataLoadingSettings(V component, S settings) {
         }
     }
 
+    protected List<? extends Grid.Column<?>> getApplicableColumns(V component) {
+        List<? extends Grid.Column<?>> componentColumns = getOrderedColumns(component);
+        List<? extends Grid.Column<?>> applicableColumns = componentColumns.stream()
+                .filter(c -> Objects.nonNull(c.getKey())).toList();
```

Comment:
This method exists to be used as a `java.util.function.Predicate`, `filter(Objects::nonNull)`. In your case, you have to call `c.getKey()` so using `Objects` makes no sense.

P.S. this is applicable to all `!isNull`

Reply:
changed Objects to operator comparison